release notes/v1.0.0.md

```diff
@@ -0,0 +1,127 @@
+k6 `v1.0.0` is here 🎉!
+
+After 9 years, 9k+ commits and 58 releases from the first commit in 2015, we're excited to announce that we're going to leave behind the `v0.x` series. Starting from now k6 aligns with the industry standard of following the semantic versioning scheme and will release this and future versions by following its new stability policy.
+
+This release includes:
+
+- Native support for extensions in the Cloud
+- New test failure control with `execution.test.fail`
+- Documentation for the stability guarantees
+
+## New features
```

Comment:
Maybe you already have this noted down else ehere, but i thought it was worth noting in a comment here that we should link to the stability guarantee document that will be merged in after the release. After reading the intro it felt like that should be either part of the intro or the first item in the New features section.

Reply:
Once we merge the [docs PR](https://github.com/grafana/k6-docs/pull/1880) for it, the link should be:

https://grafana.com/docs/k6/latest/reference/versioning-and-stability-guarantees